topic-controller/src/main/java/io/strimzi/controller/topic/ControllerAssignedKafkaImpl.java

```diff
@@ -99,37 +99,37 @@ public void changeReplicationFactor(Topic topic, Handler<AsyncResult<Void>> hand
                 fut.fail(e);
             }
         },
-        generateFuture.completer());
+            generateFuture.completer());
```

Comment:
I get why this should be indented. But shouldn't also the line 101 be indented?

Reply:
Yeah, some of the indentation rules are a little strange.